distribution/pom.xml

```diff
@@ -172,4 +166,26 @@
              </exclusions>
         </dependency>
     </dependencies>
+
+    <profiles>
+        <profile>
+            <id>doc</id>
```

Comment:
No. _dist_ is needed to include the module which contains this pom, but this profile needs to be in synch with the _doc_ profile to make sure the javadocs module it adds it the correct one.

Otherwise it might fail as there is no such valid javadoc module (say you just changed version and didn't build it), or you risk including an stale version of the documentation.

By making the dependency valid only when it's build, that also prevents Eclipse to freak out.